public/app/common/components/cc-form/style.scss

```diff
@@ -0,0 +1,42 @@
+@import "theme";
+
+.cc-form {
+  @include form-row-base();
+
+  /* fix for generated fieldset from form lib */
+  fieldset {
+    border: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  .cc-form-item {
+    @include grid-row();
+    margin: convert-to-rem(10px) 0 0 0;
+
+    .cc-form-item__label-container {
+      @include grid-column($columns: 10, $float: left);
+
+      .cc-form-item__label {
+        @include prefix-postfix-base();
+        @include prefix($bg: false, $is-button: false);
+        border: none;
+        text-align: left;
+        @include cc-label();
+
+
+      }
+    }
+    .cc-form-item__input-container {
+      @include grid-column($columns: 10);
+
+      .cc-form-item__input {
+        @include form-element();
+        background: $light-gray;
```

Comment:
We could use _Name that color_ (http://chir.ag/projects/name-that-color/) or a similar service